reconcile/utils/terrascript/models.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+from typing import Optional
+
+
+@dataclass
+class Integration:
+    name: str
+    key: str
+
+
+@dataclass
+class VaultSecret:
```

Comment:
Should this be a `Protocol` to be interchangeable with `VaultSecret` fragment from queries?

Reply:
Alternatively, you could use `reconcile.utils.secret_reader.HasSecret` -> it is already used by several integrations and complies to our convention of separating generated classes from utils libraries.

Reply:
using `HasSecret` now. Thanks for pointing that out to me.